.github/workflows/CICD.yml

```
@@ -180,10 +185,55 @@ jobs:
     name: 1️⃣ PHP tests
     needs:
       - php_syntax_errors
+    if: github.event_name != 'pull_request' || needs.changes.outputs.php == 'true'
```

actionlint (1.7.12)

[error] property "changes" is not defined in object type {php_syntax_errors: {outputs: {}; result: string}}

(expression)